tests/estimation/test_get_estimation.py

```diff
@@ -0,0 +1,122 @@
+"""
+get_estimation.py::get_estimation
+
+We test all the benchmark_mediation estimators for a certain tolerance.
+The test is skipped if estimator has not been implemented yet,
+i.e. all outputs are NaN or NotImplementedError is raised.
+The test xfails for any other wierd behavior.
+
+We pinpoint that :
+- DML_huber is not working, RRuntimeError is raised
+- multiply_robust methods return some NaN
+- A few methods get a relative error of more than 100%,
```

Comment:
change the value of beta_m and beta_t to 0.5. I suspect taking beta_t to 1 is creating some overlap issue

@judithabk6 change the names of the variables in the function to agree with the paper + change the default value

Reply:
known, open issue #8